src/usr/local/php/unraid-tailscale-utils/unraid-tailscale-utils/System.php

```
@@ -116,6 +116,43 @@ public static function checkWebgui(Config $config, string $tailscale_ipv4, bool
+
+        // This will return the PID of the tailscaled process if it is running, or an empty string if it is not.
+        $tailscaled_pid = Utils::runwrap('/usr/bin/pgrep --ns $$ --euid root -f "^/usr/local/sbin/tailscaled"', false, false);
+        if (empty($tailscaled_pid)) {
+            Utils::logwrap("tailscaled is not running.");
+            return false;
+        }
+
+        // Check if tailscaled is responsive via the local API. If it is not responsive, we will restart it.
+        $localAPI = new LocalAPI();
+        try {
+            $status = $localAPI->getStatus();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$status'. (undefined)

(UnusedLocalVariable)